README.md

```diff
@@ -36,6 +36,7 @@ These are a work in progress. None of these need to be supplied when running in
 | --- | --- | --- |
 | HTTP_PORT | 8888 | No |
 | HTTP_HOST | 0.0.0.0 | No |
+| ACCESS_TOKEN_EXPIRES_IN | 3600 | No |
```

Comment:
This is set as 3600 here, but 1800 in pom (lines 410, 520)

Reply:
I set it to 1800 in the POM just to show that the value is configurable -- the default value for the application is intended to be 3600.

Reply:
Ah...